bin/lib/zopen-build

```diff
@@ -781,10 +789,10 @@ configure()
 {
   if [ -n "${ZOPEN_CONFIGURE_CMD}" ] ; then
     printHeader "Running Configure"
-    if [ -r ${ZOPEN_ROOT}/log/config.success ]; then
-      echo "Using previous successful configuration" >&2
+    if [ -r "${ZOPEN_LOG_DIR}/config.success" ] && ! $forceRebuild; then
+      echo "Using previous successful configuration. Specify -f to force a configure." >&2
     else
-      configlog="${ZOPEN_ROOT}/log/${LOG_PFX}_config.log"
+      configlog="${ZOPEN_LOG_DIR}/${LOG_PFX}_config.log"
       create_fifo_pipe "${configlog}"
       if ! runAndLog "${ZOPEN_CONFIGURE_CMD} >$TMP_FIFO_PIPE 2>&1"; then
```

Comment:
What do you want the -f option to do?  With the code as it is, cmake will run again but will probably skip writing any new option since the existing output still exists and the source hasn't changed.  The user may not get what they want.

This will be especially apparent with the build step.  Existing .o files will be there and make will skip recompiling them even though the -f option was specified.

Reply:
created issue https://github.com/ZOSOpenTools/utils/issues/103 for this